jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/IPC.java

```diff
@@ -101,15 +104,19 @@ public class IPC {
 		String osName = props.getProperty("os.name"); //$NON-NLS-1$
 		boolean tempIsZos = false;
 		boolean tempIsWindows = false;
+		boolean tempIsLinux = false;
 		if (null != osName) {
 			if (osName.equalsIgnoreCase("z/OS")) { //$NON-NLS-1$
 				tempIsZos = true;
 			} else if (osName.startsWith("Windows")) { //$NON-NLS-1$
 				tempIsWindows = true;
+			} else if (osName.startsWith("Linux")) { //$NON-NLS-1$
+				tempIsLinux = true;
 			}
 		}
 		isZOS = tempIsZos;
 		isWindows = tempIsWindows;
+		isLinux = tempIsLinux;
```

Comment:
Please add the missing non-nls comment on line 126.

Reply:
Added